### ISSUE TITLE
Dependency states turns off

### ISSUE THREAD

**Issue by Crousinon** [maxileith/homebridge-appletv-enhanced#147]
### Analysis

When I change the volume on my Apple TV with HomePods as default speaker, all dependency states are changing to no dependency/off. When one dependency state changes, it´s correct again.

### Expected Behavior

No change when volume changes

### Steps To Reproduce

1. Connect HomePods to Apple TV
2. Play Music on Spotify or something like that > Music On is ON > Playing is ON
3. Change the volume > Music On is OFF > Playing is OFF
4. Pause the song > Music On is ON again > Playing is OFF > Paused is ON

### Logs

```logs
28/12/2023, 09:57:48] [Apple TV Enhanced] [TV ()] event devicesState: playing
28/12/2023, 09:57:48] [Apple TV Enhanced] [TV ()] New Device State Draft (might be discarded if there are state changes until the configured delay of 0ms is over): playing

**//Volume change** 

28/12/2023, 09:57:56] [Apple TV Enhanced] [TV ()] event mediaType: null
28/12/2023, 09:57:56] [Apple TV Enhanced] [TV ()] New Media Type State: music
28/12/2023, 09:57:56] [Apple TV Enhanced] [TV ()] event devicesState: null
28/12/2023, 09:57:56] [Apple TV Enhanced] [TV ()] New Device State Draft (might be discarded if there are state changes until the configured delay of 0ms is over): null  **//states are gone**

28/12/2023, 09:58:01] [Apple TV Enhanced] [TV ()] event mediaType: music
28/12/2023, 09:58:01] [Apple TV Enhanced] [TV ()] New Media Type State: music
28/12/2023, 09:58:01] [Apple TV Enhanced] [TV ()] event devicesState: paused
28/12/2023, 09:58:01] [Apple TV Enhanced] [TV ()] New Device State Draft (might be discarded if there are state changes until the configured delay of 0ms is over): paused  **//states are back**
```

### Configuration

```JSON
{
    "mediaTypes": [
        "music",
        "tv",
        "unknown",
        "video"
    ],
    "deviceStates": [
        "idle",
        "paused",
        "playing",
        "stopped"
    ],
    "remoteKeysAsSwitch": [
        "pause",
        "play",
        "stop",
        "turn_off",
        "turn_on",
        "volume_down",
        "volume_up"
    ],
    "customInputURIs": [],
    "discover": {
        "unicast": [],
        "blacklist": [],
        "multicast": true
    },
    "name": "Apple TV Enhanced",
    "deviceStateDelay": 0,
    "avadaKedavraAppAmount": 5,
    "disableVolumeControlRemote": true,  //I also tried false
    "forceVenvRecreate": false,
    "logLevel": 3,
    "platform": "AppleTVEnhanced"
}
```

### Environment

- OS: Debian GNU/Linux Bookworm (12)
- Docker:
- Homebridge: 1.7.0
- Apple TV Enhanced: 0.8.0
- Node: 18.18.2
- npm:
- Python:

**Comment by maxileith** [comment]
Debug logs would be great :)

**Comment by Crousinon** [comment]
> Debug logs would be great :). And could you please specify how you are changing the volume? Using the physical remote, the original iOS remote, the iOS remote of the plugin, a volume switch if the plugin, ...

I updated the logs. I think it doesn´t matter how I change the volume. I tried it with the tv remote, with ios remote and with your plugin.

**Comment by maxileith** [comment]
Thank you for reporting the bug.

I think you might just found a pretty major bug in a dependency that I use for this plugin. I have created a PR which should fix that bug in the dependency library: https://github.com/sebbo2002/node-pyatv/pull/291. Now we have to wait until the PR is reviewed by the owner of the library.

**Comment by maxileith** [comment]
Should be fixed in v1.0.0-3.
